app/schema_loader/schema_loader.py

```diff
@@ -4,10 +4,6 @@
 
 from app import settings
 
-import boto3
```

Comment:
I did initially remove but wanted to check today to see if it was still required. I overheard we were using boto for pushing logs to cloudwatch so it might still be required